rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -47,3 +47,18 @@ spec:
             alert_team_handle: <!subteam^S03GF42RBE2>
             team: pipelines
             runbook_url: TBD
+        - alert: CorePipelineControllerCrashloop
+          expr: |
+            sum(kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod=~"tekton-pipelines-controller-.*"}) - sum(kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod=~"tekton-pipelines-controller-.*"} offset 2m) > 0
+          for: 5m
+          labels:
+            severity: critical
+            slo: "true"
+          annotations:
+            summary: >-
+              Tekton controller is crashlooping
```

Comment:
The metric is `kube_pod_container_status_restarts_total` and the restart could happen for another reason than crashlooping. For example, the restart could be due to mem issue, OOMKilled. So, we might need to reword this. thoughts?

Reply:
you are correct here @zanssa I will reword